releases.go

```diff
@@ -230,6 +230,17 @@ func createFormation(db *gorm.DB, release *Release) error {
 	return nil
 }
 
+// GetCurrentFormations gets the current formations for an app
+func GetCurrentFormations(db *gorm.DB, app *App) (*Formation, error) {
```

Comment:
Just for consistency with the rest of the functions like this, can we un-export it and change the pluralization? (also, general convention in go is to never use "get")

```go
func currentFormation(db *gorm.DB, app *App) (Formation, error)
```